script.js

```diff
@@ -2,15 +2,295 @@ const mobileMenu = document.querySelector('#mobile-menu-modal');
 const hamburgerButton = document.getElementById('hamburger-button');
 const closeModal = document.querySelector('#modal-close-icon img');
 const mobileMenuLinks = document.querySelectorAll('.mobile-menu-link');
+const cardWorks = document.getElementById('card_works');
+const cardModel1Informations = [
+  'Multi-Post Stories',
+  "A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+  ['css', 'html', 'bootstrap', 'Ruby'],
+  'See Project',
+];
+const cardModel2Informations = [
+  'Profesional Art Printing Data',
+  "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+  ['html', 'bootstrap', 'Ruby'],
+  'See Project',
+];
+const cardItem2Title = 'Data DashboardHealthcare';
+const cardItem3Title = 'Website Protfolio ';
+
+const projectsData = {
+  workModel1: {
+    title: cardModel1Informations[0],
+    description: cardModel1Informations[1],
+    techsList: cardModel1Informations[2],
+    project: cardModel1Informations[3],
+  },
+  workModel2: {
+    title: cardModel2Informations[0],
+    description: cardModel2Informations[1],
+    techsList: cardModel2Informations[2],
+    project: cardModel2Informations[3],
```

Comment:
- [ ] Also as pointed out by the previous reviewer, it seems your pop-up is only functional on two cards. this feature is meant to be on all cards, as you will get to add all your subsequent projects to this section. As seen in your project, it seems your previous project approval had an oversight on that feature. when a card is highlighted, it should provide the `See Project` feature on all cards. you did so in one card, and that's why you can't implement it. please add the same features for all :

# Your view with no button features for all cards:
![image](https://user-images.githubusercontent.com/50990925/215986147-2ee1afdc-d066-49f8-aa76-322954e1b2da.png)

# Figma design with sample of highlighted card that shows `see project` button:
![image](https://user-images.githubusercontent.com/50990925/215986294-46443a5b-ecc0-4b49-a5ad-3aeae21cf1a4.png)

Reply:
All the see projects are shown